cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteServiceManager.java

```diff
@@ -40,10 +42,10 @@
   private final ServiceClient serviceClient;
   private final Id.Service serviceId;
 
-  public RemoteServiceManager(Id.Service programId, ClientConfig clientConfig, RESTClient restClient,
+  public RemoteServiceManager(ProgramId programId, ClientConfig clientConfig, RESTClient restClient,
```

Comment:
serviceId is of class `Id.Service`. There are some API's still using that and not necessary to change for the app versioning ITN.